projects/dotcms-field-elements/src/components/dot-tags/components/dot-chip/dot-chip.e2e.ts

```diff
@@ -4,105 +4,101 @@ describe('dot-chip', () => {
+            it('should render', async () => {
+                element.setProperty('deleteLabel', 'Remove');
+                await page.waitForChanges();
 
-            const button = await element.find('button');
-            expect(button.innerHTML).toBe('x');
-        });
+                const button = await getButton();
+                expect(button.innerText).toBe('Remove');
+            });
 
-        it('should be disabled', async () => {
-            element.setAttribute('disabled', true);
-            await page.waitForChanges();
+            it('should render default', async () => {
+                await page.waitForChanges();
 
-            const button = await element.find('button');
-            expect(button.getAttribute('disabled')).not.toBeNull();
+                const button = await getButton();
+                expect(button.innerText).toBe('Delete');
+            });
         });
 
-        describe('render with invalid attributes', () => {
-            it('should render with no string in deleteLabel', async () => {
-                element.setAttribute('delete-label', {});
+        describe('disabled', () => {
+            it('should render', async () => {
+                element.setProperty('disabled', true);
                 await page.waitForChanges();
 
-                const button = await element.find('button');
-                expect(button.innerHTML).toBe('[object Object]');
+                const button = await getButton();
+                expect(button.getAttribute('disabled')).toBeDefined();
             });
 
-            it('should be disabled', async () => {
-                element.setAttribute('disabled', {});
+            it('should render default', async () => {
                 await page.waitForChanges();
 
-                const button = await element.find('button');
-                expect(button.getAttribute('disabled')).not.toBeNull();
+                const button = await getButton();
+                expect(button.getAttribute('disabled')).toBeNull();
             });
         });
     });
 
-    describe('events', () => {
+    describe('@Events', () => {
+        let spyRemoveEvent;
+
         beforeEach(async () => {
             page = await newE2EPage({
-                html: `<dot-chip
-                            label='test-tag'
-                            disabled>
-                        </dot-chip>`
+                html: '<dot-chip label="test-tag"></dot-chip>'
             });
 
             element = await page.find('dot-chip');
+            spyRemoveEvent = await element.spyOnEvent('remove');
+            await page.waitForChanges();
         });
 
-        it('should trigger remove event', async () => {
-            const spyRemoveEvent = await element.spyOnEvent('remove');
+        describe('remove', () => {
+            it('should trigger', async () => {
+                const button = await getButton();
+                button.triggerEvent('click');
```

Comment:
add `await`

Reply:
<img width="711" alt="Screen Shot 2019-06-04 at 3 50 07 PM" src="https://user-images.githubusercontent.com/751424/58916263-72e64280-86e0-11e9-8378-5be24827af51.png">

triggerEvent doesn't return a promise so we don't need the await, but in this case I change it to a `.click()`.